verilog/preprocessor/verilog_preprocess.cc

```diff
@@ -552,6 +553,134 @@ absl::Status VerilogPreprocess::HandleEndif(
   return absl::OkStatus();
 }
 
+// Handle `include directives.
+absl::Status VerilogPreprocess::HandleInclude(
```

Comment:
Did you have a look how the kythe extractor and/or project_tool (I forgot where ti is implemented) does the include handling ?
Ideally we keep the ownership of the full file in one place (and then the string ownership where the tokens are pointing to is clear).

Reply:
No, I will try to find it.